import_busstops.py

```diff
@@ -6,33 +6,39 @@
 class BusImportCommand(CSVImportCommand):
 
     def process_row(self, row):
-        bus_stop = {
-            "amic_code": row[0],
-            "point": {
-                "type": "Point",
-                "coordinates": [float(row[29]), float(row[30])]
-            },
-            "name": row[4],
-            "direction": row[14],
-            "area": row[19],
-            "road": row[10],
-            "nptg_code": row[1],
-            "srid": 4326
-        }
-
-        headers = {'Authorization': 'Token {0}'.format(self.token)}
-
-        response = requests.post(
-            self.api_url,
-            json=bus_stop,
-            headers=headers)
-
-        if response.status_code == 201:
-            print('{0} imported correctly'.format(row[0]))
-        else:
+        try:
+            # Import the record only if it's one of the possible bus stops
+            if row[31] in ['BCT', 'BCE', 'BST', 'BCS', 'BCQ']:
```

Comment:
Could you make and example of how you would assert this, please? Column name is unknown in this script because we are analysing a single row. In the main class we have instead access to the header and we skip it, according to the parameter we pass to the base class.